### ISSUE TITLE
Wrong configuration in Testing Single-File Components with Karma

### ISSUE THREAD

**Issue by Juraci** [vuejs/vue-test-utils#1225]
### Version
1.0.0-beta.29

### Reproduction link
[https://github.com/vuejs/vue-test-utils/blame/v1.0.0-beta.29/docs/guides/testing-single-file-components-with-karma.md#L37](https://github.com/vuejs/vue-test-utils/blame/v1.0.0-beta.29/docs/guides/testing-single-file-components-with-karma.md#L37)

### Steps to reproduce
1. Create a new Vue project with @vue/cli@3.x.x without any testing options
1. Follow the steps to setup karma testing [here](https://github.com/vuejs/vue-test-utils/blob/v1.0.0-beta.29/docs/guides/testing-single-file-components-with-karma.md#testing-single-file-components-with-karma)
2. npm run test

### What is expected?
Karma should tell you that there are no specs to run.

### What is actually happening?
Error messaging telling that `./webpack.config.js` doesn't exit.

---
As per @vue/cli@3.x.x the webpack.config.js file is no longer created in the root folder.

This part of the documentation will lead to a broken karma setup: [link](https://github.com/vuejs/vue-test-utils/blame/v1.0.0-beta.29/docs/guides/testing-single-file-components-with-karma.md#L37)

**Comment by eddyerburgh** [comment]
Yes we should update the docs considering that we recommend `@vue/cli` instead of `vue-cli`.
